code/model/modelrender.h

```diff
@@ -97,6 +97,8 @@ class model_render_params
 
 	bool Normal_extrude;
 	float Normal_extrude_width;
+
+	model_render_params(const model_render_params& other);
```

Comment:
You should use `= delete` and also delete the copy operator to fully disable copying.